packages/browser-sdk/src/lib/react/Grid/index.tsx

```diff
@@ -33,34 +61,38 @@ interface GridVideoCellViewProps {
     participant: CellView["client"];
     isPlaceholder?: boolean;
     isSubgrid?: boolean;
-    render?: () => React.ReactNode;
+    render?: ({ cell }: { cell: GridCell }) => React.ReactNode;
     onSetClientAspectRatio: ({ aspectRatio, clientId }: { aspectRatio: number; clientId: string }) => void;
 }
 
-function GridVideoCellView({ aspectRatio, participant, render }: GridVideoCellViewProps) {
+function GridVideoCellView({ aspectRatio, participant, onSetClientAspectRatio, render }: GridVideoCellViewProps) {
     const videoEl = React.useRef<WherebyVideoElement>(null);
 
     const handleResize = React.useCallback(() => {
         const ar = videoEl.current && videoEl.current.captureAspectRatio();
 
         if (ar && ar !== aspectRatio && participant?.id) {
-            // onSetClientAspectRatio({ aspectRatio: ar, clientId: participant.id });
+            onSetClientAspectRatio({ aspectRatio: ar, clientId: participant.id });
         }
     }, []);
 
     return (
         <div>
-            {render ? (
-                render()
-            ) : participant?.stream && participant.isVideoEnabled ? (
-                <VideoView
-                    ref={videoEl}
-                    stream={participant.stream}
-                    onVideoResize={handleResize}
-                    style={{
-                        borderRadius: "8px",
-                    }}
-                />
+            {participant?.stream && participant.isVideoEnabled ? (
+                <>
+                    {render ? (
+                        render()
```

Comment:
Should we pass the cell in here like it says on line 64?

Reply:
Ah I see you add it in a later commit

Reply:
This was changed in a later commit. It might be easier to review the full changes, as there are some changes between the commits. Sorry about that!

Reply:
All good!